tests/framework/matcher/existence.go

```diff
@@ -31,12 +32,36 @@ func (e existMatcher) Match(actual interface{}) (success bool, err error) {
 	return true, nil
 }
 
+func formatObject(actual interface{}) string {
+	obj := reflect.ValueOf(actual)
+	if obj == (reflect.Value{}) || helper.IsNil(actual) || helper.IsSlice(actual) {
```

Comment:
`obj == (reflect.Value{})` might be redundant, since `reflect.ValueOf(actual)` returns `reflect.Value{}` when `actual` is `nil` and that is already checked by `helper.IsNil(actual)`.